subsys/fs/pm.yml

```diff
@@ -0,0 +1,7 @@
+#include <autoconf.h>
+
+#ifdef CONFIG_FILE_SYSTEM_NFFS
+nffs_storage:
+  placement: {after: [mcuboot_storage, app]}
```

Comment:
--mcuboot_storage, elsewhere

Reply:
This will actually not break if mcuboot_storage is removed (and at this point it is not, as that PR is blocked by this). 

I will make these fixes in https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/pull/65